backend/src/service/wish.service.ts

```diff
@@ -0,0 +1,25 @@
+import { DeleteResult } from 'typeorm';
+import { INVALID_ACCESS } from '../constants/client-error-name';
+import { NotFoundError } from '../errors/client.error';
+import { WishRepository } from '../repository/wish.repository';
+import { CreateWishResponse } from '../types/response/wish.response';
+
+async function createWish(userId: number, goodsId: number): Promise<CreateWishResponse> {
+  return await WishRepository.createWish(userId, goodsId);
+}
+
+async function deleteWish(userId: number, goodsId: number): Promise<DeleteResult> {
+  await isMineWish(userId, goodsId);
+  return await WishRepository.deleteWish(userId, goodsId);
+}
+
+async function isMineWish(userId: number, goodsId: number): Promise<boolean> {
+  const wish = await WishRepository.findWishByIds(userId, goodsId);
+  if (wish) return true;
+  throw new NotFoundError(INVALID_ACCESS);
```

Comment:
return boolean이 필요없어보입니다!

```suggestion
  if (!wish) throw new NotFoundError(INVALID_ACCESS);
```

Reply:
그렇네요! 바로 적용했습니다.  👏